code/renderers/vue3/src/docs/extractArgTypes.ts

```diff
@@ -44,10 +44,11 @@ export const extractArgTypes: ArgTypesExtractor = (component) => {
       // skip duplicate and global props
       if (!argType || argTypes[argType.name]) return;
 
-      argTypes[argType.name] = {
-        ...argType,
-        control: { disabled: !['props', 'slots'].includes(section) },
```

Comment:
We still require storybook controls for slots. Indeed, it does not cover all cases, not due to Vue's capabilities, but because we do not yet have the compatible type of control.  Currently, we have a basic one, but adding this feature involves intervention and opinions from other team members. We had plan me and @kasperpeulen to collaborate on this in future releases.